utils/sentry_client.py

```diff
@@ -89,6 +97,20 @@ def optional_project_fields(self):
         }
         return optional_fields
 
+    def validate_project_options(self, options):
+        # If the resolve age is not one of these then sentry will set to 0
+        valid_resolve_age = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 15, 18, 21,
+                             24, 30, 36, 48, 72, 96, 120, 144, 168, 192, 216,
+                             240, 288, 312, 336, 360, 384, 408, 432, 456, 480,
+                             504, 528, 552, 576, 600, 624, 648, 672, 696, 720]
```

Comment:
Where did you get these numbers from? Maybe a comment on that to make the maintenance easier in the future?